recipe/thirdparty/emaildelivery/smtpService/serviceImplementation.go

```diff
@@ -0,0 +1,30 @@
+package smtpService
```

Comment:
You do not need this as you can use the email verification recipe's smtp service implementation directly (since third party's emails are exactly the same type as email verification emails)